### DISCUSSION TITLE
Which is max lenght response i can config to the bot?

### DISCUSSION THREAD

**Discussion by doctorerre** [mishl-dev/Discord-AI-Chatbot#55] [General]
Really liked the project. I know discord if u dont use nitro admits 2300 char max .... is the same lenght i can config the bot?

**Comment by doctorerre** [comment]
Ah ok, the bot gave me the answer xd

As an AI language model, I can generate responses of varying lengths depending on the complexity and nuance of the input. However, in a platform like Discord without Nitro, the maximum length of a single message is 2000 characters. So, my responses may need to be broken up into multiple messages if they exceed that limit.

**Reply by mishl-dev** [reply]
yeaa
